ironfish/src/rpc/routes/wallet/exportAccount.ts

```diff
@@ -13,6 +13,7 @@ export type ExportAccountResponse = {
     incomingViewKey: string
     outgoingViewKey: string
     publicAddress: string
+    version: number
   }
```

Comment:
You can reset all the files for the `version` changes since they are all in different PR now